scripts/install/etc/sysconfig/motr

```diff
@@ -63,9 +63,17 @@ MOTR_M0T1FS_MOUNTPOINT='/mnt/m0t1fs'
 MOTR_M0D_MAX_RPC_MSG_SIZE=524288
 
 # Minimum rpc receive queue length.
+# Packets from clients will be dropped if the queue is not big enough,
+# and this might affect the performance dramatically. Increase this
+# (for example, to 512) to avoid packets drops on receive.
 #MOTR_M0D_MIN_RPC_RECVQ_LEN=16
 
-# ioservice buffer pool configuration
+# ioservice buffer pool configuration.
+# This parameter relates to the above MOTR_M0D_MIN_RPC_RECVQ_LEN, it
```

Comment:
This is not right, one RPC message may require many zero copy buffers.
Both are independent.

Reply:
They are independent, but they are related also, aren't they? And I tried to explain, how they are related. Please, suggest a better text which will be more clear.

Reply:
They are not related.
IOS and SNS buffers are independent, impact IO Writes/Reads and SNS IO only.

Reply:
Ok, I see it is confusing, so let me drop this "related" note.